packages/react/src/components/popover/popover.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Alignment,
+  c,
+  classy,
+  m,
+  PopoverProps as BaseProps,
+  Position,
+} from '@onfido/castor';
+import React, { ReactNode, RefObject, useRef, useState } from 'react';
+import { createPortal } from 'react-dom';
+import { withRef } from '../../utils';
+import { useObserver } from './useObserver';
+import { useOnClickOutside } from './useOnClickOutside';
+
+export interface PopoverProps extends BaseProps, Omit<Div, 'ref'> {
+  children?: ReactNode;
+  onClose?: () => void;
+  target?: RefObject<Element>;
+}
+
+export const Popover = ({
```

Comment:
Curious, why did you choose verb-y `align` and `place` rather than noun-y `alignment` and `placement`?

Reply:
Mostly because it's terser, but also because they exist in CSS like `align-items` and `place-items`.
One could argue that `place` represents `justify + align` but we don't `justify` the popover.

Perhaps `position & align` works better, which is pretty much how Popover relates to CSS.
i.e. `position: top; align-items: start` (even though the implementation doesn't use Flex or Grid).

Happy to go with nouns too.
Naming is hard.

Reply:
Your call on this 😏

Reply:
Went with `position & align`.
There's still time to change before releasing.